src/components/common/CreatorCard.tsx

```diff
@@ -129,9 +134,30 @@ const CreatorCard: React.VFC<{
 
   const tabsContents = settings['creator.content.tab']?.split(',') || []
   const hasTabSetting = Boolean(settings['creator.content.tab'])
+  const tabsWithCount =
+    settings['creator.content.tab.with_count'] && Boolean(JSON.parse(settings['creator.content.tab.with_count']))
+  const {
+    isLoading: getAggregateByMemberIdLoading,
+    programCount,
+    podcastProgramCount,
+  } = useGetAggregateByMemberIdAndRoleName(id, 'instructor')
+  const { isLoading: getAggregateByCreatorIdLoading, appointmentPlanCount } = useGetAggregateByCreatorId(id)
```

Comment:
`useGetAggregateByCreatorId` is ambiguous, may be could be `useGetAppointmentPlanAggregateByCreatorId`